src/core/simplify.rkt

```diff
@@ -17,16 +17,6 @@
   (->* (expr? #:rules (listof rule?)) (#:precompute boolean?) expr?)
   (last (first (simplify-batch (list expr) #:rules rls #:precompute precompute?))))
 
-
-(define (get-proof input start end)
-  (run-simplify-input
-    input
-    (lambda (egg-graph node-ids iter-data)
-      (define proof (egraph-get-proof egg-graph start end))
-      (when (null? proof)
-        (error (format "Failed to produce proof for ~a to ~a" start end)))
-      proof)))
-
```

Comment:
We don't call this? I guess it's just a convenience method? I'm fine deleting it, just making sure.

Reply:
Ah, I see it moved. Did you make any changes or just move it?